src/nautilus_librarian/typer/commands/workflows/actions/auto_commit_base_images.py

```diff
@@ -91,6 +100,29 @@ def commit_deleted_base_image(
     )
 
 
+def commit_renamed_base_image(
+    git_repo_dir,
+    old_base_img_relative_path,
+    new_base_img_relative_path,
+    gnupghome,
+    git_user,
+):
+    repo = GitRepo(git_repo_dir, git_user, gnupghome)
+
+    return repo.commit(
+        {
+            "renamed": {
+                "old": files_to_commit(old_base_img_relative_path),
```

Comment:
I think it was me who created and named that function (`files_to_commit`) but I do not like it anymore because we have to know DVC add implementation details (the files DVC add generates). I think we could refactor it this way:

```
files_added_to_git = dvc_api_wrapper.add(base_img_relative_path)
dvc_api_wrapper.push(f"{base_img_relative_path}.dvc")

commit_base_image(git_repo_dir, base_img_relative_path, gnupghome, git_user)
```

```
def commit_base_image(git_repo_dir, files_added_to_git, gnupghome, git_user):
    repo = GitRepo(git_repo_dir, git_user, gnupghome)

    return repo.commit(
        files_added_to_git,
        commit_message=f"feat: new base image: {os.path.basename(base_img_relative_path)}",
    )
```

This way we do not need to know the internals of DVD add.

Reply:
It makes totally sense.

Two comments about it:
 - Keep in mind that the delete and rename methods of the auto-commit does not call to any DVC Wrapper method, as all DVC-related operation is done in the "rename image" and "delete image" themselves. So instead of retrieving the files to commit via return value of the DVC wrapper add function, I'm going to move into the wrapper the "get_files_to_commit" method, so it can be used for any DVC action to a file that is to be committed afterwards.
 - There are other nasty couplings between the actions and the DVC internals. The push method, for instance, requires as a parameter the pointer and not the file. I think that we must check for those and fix them.